src/Sylius/Bundle/CoreBundle/Form/Type/ProductReviewAdminType.php

```diff
@@ -21,27 +20,14 @@
  * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
  * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
  */
-class ProductReviewAdminType extends ProductReviewType
+class ProductReviewAdminType extends ReviewType
 {
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         parent::buildForm($builder, $options);
-
-        $builder
-            ->add('reviewSubject', 'entity', [
-                'class' => Product::class,
-                'label' => 'sylius.form.review.product',
-                'property' => 'name',
-            ])
-            ->add('author', 'entity', [
-                'class' => Customer::class,
-                'label' => 'sylius.form.review.author',
-                'property' => 'email',
-            ])
-        ;
```

Comment:
Generally, we don't need this, but I left that because we also have empty `ProductReviewType`. What do you think? Should I remove them and use basic `ReviewType`?